pkg/appconsts/versioned_consts_test.go

```diff
@@ -72,6 +72,30 @@ func TestVersionedConsts(t *testing.T) {
 			expectedConstant: v3.MaxTxSize,
 			got:              appconsts.MaxTxSize(v3.Version),
 		},
+		{
```

Comment:
doesn't have to be addressed in this pr but we probably need to come up with a more efficient way of testing this before it gets out of hand with all the versions